lector/resources/raw/io.github.BasioMeusPuga.Lector.metainfo.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 BasioMeusPuga <https://github.com/BasioMeusPuga> -->
+<!-- See <https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html> for an overview of the AppStream metainfo.xml spec. -->
+<component type="desktop-application">
+    <id>io.github.BasioMeusPuga.Lector</id>
```

Comment:
The AppStream specification requires that the ID follow a "reverse-DNS" scheme (like an Internet domain name with the domain components backwards) to make sure the ID is totally unique. Since Lector doesn't have its own website, I based it on what the GitHub Pages domain might be like if you ever created a GH Pages site -- but with a dot for another "subdomain" at the end, since this scheme doesn't allow slashes like "/Lector/"...

But you could just as well do this based on the URL for your personal blog or whatever, if you have one.